compiler/z/codegen/BinaryCommutativeAnalyser.cpp

```diff
@@ -741,8 +742,12 @@ TR_S390BinaryCommutativeAnalyser::integerAddAnalyser(TR::Node * root, TR::InstOp
       is16BitMemory2Operand = true;
       }
 
-   /**  Attempt to use AGH to add halfworf from memory */
-   if (cg()->comp()->target().cpu.getSupportsArch(TR::CPU::z14) &&
+   /*
+    *  Attempt to use AGH to add halfworf from memory.
+    *  Ensure that Misc. Instructions Extension Facility 2
+    *  is installed on the machine.
+    */
```

Comment:
"halfworf" seems to be an existing typo. Let's correct it while we're in this area. I don't think the rest of the comment is too useful (here nor in the location above) as it is obvious from the API call in the `if` that we require MIE2 facility.

Reply:
Fixed in the latest commit - 83cbffa